tensorboard/webapp/settings/BUILD

```diff
@@ -20,6 +20,8 @@ ng_module(
         "//tensorboard/webapp/angular:expect_angular_material_icon",
         "//tensorboard/webapp/angular:expect_angular_material_input",
         "//tensorboard/webapp/core",
+        "//tensorboard/webapp/core/store",
+        "//tensorboard/webapp/core/actions",
```

Comment:
some buildifier on these files would be nice (I used it before so it is kosher here; this file requires alphasorting).

Reply:
I'd really like to get buildifier working externally as a matter of course (i.e., even in our CI).  Let's punt this until then.